tka/sig.go

```diff
@@ -74,6 +84,9 @@ type NodeKeySignature struct {
 	//
 	// Intermediate SigRotation signatures may omit this value to use the
 	// parent one.
+	//
+	// SigCredential signatures use this field to specify the public key
```

Comment:
Should we change the name of the field to not refer to rotation then?

`IntermediatePubkey` or something?

Reply:
Ive gone with `WrappingPubkey`, wdyt?